src/core_modules/capture-core/components/Pages/MainPage/EventWorkingLists/UpdateTrigger/eventWorkingListsUpdateTrigger.types.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import type { LoadView, UpdateList } from '../../WorkingLists';
+import type { EventWorkingListsRowMenuSetupOutputProps } from '../RowMenuSetup';
+
+type ExtractedProps = $ReadOnly<{|
+    lastTransaction: number,
+    lastIdDeleted?: string,
+    listDataRefreshTimestamp?: number,
+    lastTransactionOnListDataRefresh?: number,
+    onLoadView: Function,
+    onUpdateList: Function,
+|}>;
+
+type RestProps = $Rest<EventWorkingListsRowMenuSetupOutputProps & { lastIdDeleted: string, lastTransactionOnListDataRefresh: number, listDataRefreshTimestamp: number },
+    ExtractedProps & { lastIdDeleted: string, lastTransactionOnListDataRefresh: number, listDataRefreshTimestamp: number }>;
+
+export type Props = $ReadOnly<{|
+    ...RestProps,
+    ...ExtractedProps,
+|}>;
+
+export type EventWorkingListsUpdateTriggerOutputProps = $ReadOnly<{|
+    ...RestProps,
+    customUpdateTrigger: string,
+    forceUpdateOnMount: boolean,
+    onLoadView: LoadView,
+    onUpdateList: UpdateList,
+|}>;
```

Comment:
So, the solution with the OutputProps is what makes sense to me

Reply:
Apologies for the confusion. I wasnt implying that I would prefer you to annotate 50 props. 

I am only asking if it makes sense for the `EventWorkingListsUpdateTriggerOutputProps` to be defined in the `WorkingListsInterfaceBuilder/workingListsInterfaceBuilder.types.js`. The WorkingListsInterfaceBuilder module is the only module that consuming this type therefore I think it would make sense for us to have it defined there. 

Now we define it inside the `UpdateTrigger` and we export it from here to import it in the `WorkingListsInterfaceBuilder`. We can instead export the `RestProps`.

Reply:
That doesn't work because `UpdateTrigger` passes on props that `WorkingListsInterfaceBuilder` potentially doesn't care about.

Hope we can agree this is not a blocker and move on.